tests/Test/WebTestCaseTest.php

```diff
@@ -0,0 +1,285 @@
+            ->method('loadUserByUsername')
+            ->willReturn($user);
+
+        $session = $this->getMock('stdClass', ['getId', 'getName', 'set', 'save']);
+
+        $this->container
+            ->method('get')
+            ->withConsecutive(['beelab_user.manager'], ['session'])
+            ->will($this->onConsecutiveCalls($repository, $session));
+
+        $cookieJar = $this->getMock('stdClass', ['set']);
+
+        $this->client
+            ->method('getCookieJar')
+            ->willReturn($cookieJar);
+
+        // Call `login` method
+        $method = new \ReflectionMethod($mock, 'login');
+        $method->setAccessible(true);
+        $method->invoke($mock);
+    }
+
+    public function testGetFile()
+    {
+        $mock = $this->getMock('Beelab\TestBundle\Test\WebTestCase', null);
+
+        // Call `getFile` method
+        $method = new \ReflectionMethod($mock, 'getFile');
+        $method->setAccessible(true);
+        $file = $method->invoke($mock, 'test', 'text', 'txt', 'text/txt');
+
+        $this->assertInstanceOf(UploadedFile::class, $file);
+    }
+
+    public function testGetImageFile()
+    {
+        $mock = $this->getMock('Beelab\TestBundle\Test\WebTestCase', null);
+
+        // Call `getImageFile` method
+        $method = new \ReflectionMethod($mock, 'getImageFile');
+        $method->setAccessible(true);
+        $file = $method->invoke($mock);
+
+        $this->assertInstanceOf(UploadedFile::class, $file);
+    }
+
+    public function testGetPdfFile()
+    {
+        $mock = $this->getMock('Beelab\TestBundle\Test\WebTestCase', null);
+
+        // Call `getPdfFile` method
+        $method = new \ReflectionMethod($mock, 'getPdfFile');
+        $method->setAccessible(true);
+        $file = $method->invoke($mock);
+
+        $this->assertInstanceOf(UploadedFile::class, $file);
+    }
+
+    public function testGetZipFile()
+    {
+        $mock = $this->getMock('Beelab\TestBundle\Test\WebTestCase', null);
+
+        // Call `getZipFile` method
+        $method = new \ReflectionMethod($mock, 'getZipFile');
+        $method->setAccessible(true);
+        $file = $method->invoke($mock);
+
+        $this->assertInstanceOf(UploadedFile::class, $file);
+    }
+
+    public function testGetTxtFile()
+    {
+        $mock = $this->getMock('Beelab\TestBundle\Test\WebTestCase', null);
+
+        // Call `getTxtFile` method
+        $method = new \ReflectionMethod($mock, 'getTxtFile');
+        $method->setAccessible(true);
+        $file = $method->invoke($mock);
+
+        $this->assertInstanceOf(UploadedFile::class, $file);
+    }
+
+    public function testLoadFixtures()
+    {
+        $mock = $this->getMock('Beelab\TestBundle\Test\WebTestCase', ['getContainer', 'loadFixtureClass']);
+
+        $mock
+            ->method('getContainer')
+            ->willReturn($this->container);
+
+        $mock
+            ->expects($this->exactly(2))
+            ->method('loadFixtureClass');
```

Comment:
Why this does not work?
The mock calls the original loadFixtureClass method making to fail the test.